docs/quickstart/classification.md

```diff
@@ -99,6 +99,27 @@ plot.confusion_matrix(y_test, y_pred_dt)
 plot.feature_importances(dtc, top_n=5, feature_names=list(dtc.feature_names_in_))
 ```
 
+Use the predicted labels or classes to plot an ROC curve.
+We support 3 main formats that allow you to quickly generate a single or a multi classification ROC curve.
+
+```
+array([0, 1, 2])
+
+array([[0, 1, 0], 
+        [1, 0, 0], 
+        [0, 0, 1]])
+
+array([[0.1, 0.8, 0.1], 
+        [0.8, 0.1, 0.1], 
+        [0.1, 0.1, 0.8]])
+```
+
+For this example we will plot a multi classification ROC using the predicted classes 
+
+```{code-cell} ipython3
+plot.roc(y_test, y_pred_dt)
```

Comment:
there's an error here, y_pred_dt is created with `y_pred_dt = dt_model.predict(X_test)`

but it should be `y_pred_dt = dt_model.predict_proba(X_test)`

note that some plots like confusion matrix need the output of `predict`, and others like roc need `predict_proba`